src/InstalledPackage.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
Just from the commit message alone, I think that one thing we need in the /docs is a glossary so that we have ubiquitous language.